build.sbt

```diff
@@ -3,6 +3,9 @@ organization in ThisBuild := "sample.helloworld"
 // the Scala version that will be used for cross-compiled libraries
 scalaVersion in ThisBuild := "2.11.7"
 
+// the ConductR Developer Sandbox version that will be used to start the ConductR cluster
+SandboxKeys.imageVersion in Global := "1.1.2"
```

Comment:
You get a very polite message:

```
> sandbox run
[error] (*/*:conductrSandboxRun) No mapping for imageVersion. imageVersion must be set. Please visit https://www.typesafe.com/product/conductr/developer for current version information.
```

> On 15 Mar 2016, at 12:26 PM, James Roper <notifications@github.com> wrote:
> 
> In build.sbt <https://github.com/typesafehub/activator-lagom-java/pull/5#discussion_r56103414>:
> 
> > @@ -3,6 +3,9 @@ organization in ThisBuild := "sample.helloworld"
> >  // the Scala version that will be used for cross-compiled libraries
> >  scalaVersion in ThisBuild := "2.11.7"
> >  
> > +// the ConductR Developer Sandbox version that will be used to start the ConductR cluster
> > +SandboxKeys.imageVersion in Global := "1.1.2"
> So, if we don't define it, what happens when you run the project?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/typesafehub/activator-lagom-java/pull/5/files#r56103414 <https://github.com/typesafehub/activator-lagom-java/pull/5/files#r56103414>